packages/cli/test/unit/commands/env.test.ts

```diff
@@ -205,10 +205,17 @@ describe('env', () => {
       });
       const cwd = setupUnitFixture('vercel-env-pull');
       client.cwd = cwd;
-      client.setArgv('env', 'pull', 'other.env', '--yes');
+      client.setArgv(
+        'env',
+        'pull',
+        'other.env',
+        '--yes',
+        '--environment',
+        'production'
+      );
       const exitCodePromise = env(client);
       await expect(client.stderr).toOutput(
-        'Downloading `development` Environment Variables for Project vercel-env-pull'
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** Add assertion (or new test) somewhere that development env pulls do not include system env vars at all.